app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -4,7 +4,7 @@ def do_clone_task_check(clone_task_ref)
       instance = openstack.handled_list(:servers).detect { |s| s.id == clone_task_ref }
       status   = instance.state.downcase.to_sym
 
-      return true if status == :active
+      return true if [:active, :error].include?(status)
```

Comment:
@mansam If we know there is an error you could raise an `MiqException::MiqProvisionError` which would get processed by the state machine here (https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request_task/state_machine.rb#L23).

This would be better than returning `true` and then having `poll_destination_in_vmdb` moving into the `customize_destination` state.